### DISCUSSION TITLE
Creator signature verification questions/issues

### DISCUSSION THREAD

**Discussion by l8nit3tr0ubl3** [metaplex-foundation/metaplex#1721] [Q&A] [answered]
I have created and upload my project (1337 pieces) and have minted on test/creators NFT to ensure the mint process works.
Once I had minted the NFT, I went on to attempt to use the sign function to sign with the creating wallet. Each attempt listed 1337 NFTs , however 0 to be signed by the actual creating wallet.

My issue, i believe , is when creating the initial template for my collection, I set 3 different "royalties" addresses (a.k.a creator addresses) and did NOT use the "creator pubkey" as any of the 3 royalties recievers. I did not realize that these addresses were needed to sign the metadata afterwards.

once importing one of those 3 into my solana CLI, i was able to sign the one minted NFT.

```
Creator pubkey:  9R4z3YnHBRmMk5btNz8EGbkTFWbPjL8qfVSgwKXg8Su7
Environment:  mainnet-beta
Candy machine address:  CzqyTm6PcguRKo5kGtSyc9jeLwNxT8fgG9RjJ9Joqa88
Batch Size:  1
Found 1 nft's minted by candy machine Fk9AqZWAqWZBV65MfZhaARm9bUbJccUzLjzmdoKKakQf
Found 0 nft's to sign by  9R4z3YnHBRmMk5btNz8EGbkTFWbPjL8qfVSgwKXg8Su7
```

My question:
Is there a way to o the signing process with the original keypair that actually generated/uploaded the project and candy machine? Even though it is not listed in the metadata as a receiver of the royalties?

Or must I now import the wallet for the 3 pub keys ive have listed in my original template?

TL;DR
Does the creating wallet need to be listed as 'creator' in template even if it does not receive royalties?

**Comment by l8nit3tr0ubl3** [comment, marked as answer]
For anyone running into this, you must locate the any of your minted NFT collection on solscan (or other explorer) and click 'creators'.
The first address (inserted by metaplex on upload i believe)is the one used when trying to create a mint ID list for magic eden.

ISSUE RESOLVED
